src/breadcrumb/breadcrumb.stories.ts

```diff
@@ -59,6 +61,25 @@ storiesOf("Breadcrumb", module)
 		items: createBreadcrumbItems
 	}
 }))
+.add("Model with templates", () => ({
+	template: `
+	<ng-template #breadcrumbTemplate let-item>
+		Hello {{ item.content }}
```

Comment:
Maybe make this a knob that allows them to change what's in the template

Reply:
Yeah... could also make the `item.content` configurable as well 🤔